solarforecastarbiter/validation/tests/test_validator.py

```diff
@@ -227,13 +225,64 @@ def test_check_poa_clearsky(mocker, times):
     assert_series_equal(result, expected)
 
 
-def test_check_irradiance_day_night():
+def test_check_day_night():
     MST = pytz.timezone('MST')
     times = [datetime(2018, 6, 15, 12, 0, 0, tzinfo=MST),
              datetime(2018, 6, 15, 22, 0, 0, tzinfo=MST)]
     expected = pd.Series(data=[True, False], index=times)
     solar_zenith = pd.Series(data=[11.8, 114.3], index=times)
-    result = validator.check_irradiance_day_night(solar_zenith)
+    result = validator.check_day_night(solar_zenith)
+    assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize('closed,expected', (
+    ('left', pd.Series([False, True], index=pd.DatetimeIndex(
+        ['20200917 0000', '20200917 0100']))),
+    ('right', pd.Series([False, True], index=pd.DatetimeIndex(
+        ['20200917 0100', '20200917 0200'])))
+))
+def test_check_day_night_interval(closed, expected):
+    interval_length = pd.Timedelta('1h')
+    index = pd.date_range(
+        start='20200917 0000', end='20200917 0200', closed=closed,
+        freq='5min')
+    solar_zenith = pd.Series([89]*11 + [80]*13, index=index)
+    result = validator.check_day_night_interval(
+        solar_zenith, closed, interval_length
+    )
+    assert_series_equal(result, expected)
```

Comment:
It doesn't add any coverage, but adding a test with the same parameters and `solar_zenith= pd.Series([89]*10 + [80]*14, index=index)`  where expected = [True, True]  to test just below/above the thresh hold would be nice to reinforce the expected behavior.